### ISSUE TITLE
fomm vs adv

### ISSUE THREAD

**Issue by nagadit** [AliaksandrSiarohin/motion-cosegmentation#35]
What's the difference in using `--checkpoint vox-adv-cpk.pth.tar` and --checkpoint vox-first-order.pth.tar checkpoints with --config config/vox-256-sem-10segments.yaml ?

The vox-adv-cpk.pth.tar includes vox-first-order.pth.tar?

I see no difference.

**Comment by AliaksandrSiarohin** [comment]
Yes vox adv is first order (keypoint base model). 10 segments is a segmentation based model.
